### PR TITLE
GPIB-593

### PR DESCRIPTION
**Pull request by xiangzhe0812**

### CODE REVIEW

File: src/pages/ProfilePage.js [hunk 1]
```diff
@@ -158,6 +158,30 @@ const Dashboard = () => {
...
+  const syncBankAccount = async () => {
+    setSyncingBankAccount(true);
+    try {
+      await gpib.secure.post("/xero/syncBankAccount");
+    } catch (error) {
+      console.error(error);
+      setErrorMessage(error);
+      setSyncingBankAccount(false);
+    }
+    setSyncingBankAccount(false);
```
Comment (Mitchell139) [inline src/pages/ProfilePage.js]: I'm likely wrong but wouldn't this be run twice if it runs the catch?
```suggestion
    }
    setSyncingBankAccount(false);
```

**Review by Mitchell139 — COMMENTED** [review]
FWIW looks alright to me

File: src/pages/ProfilePage.js (hunk 1 above)
Comment (bitcoinbrisbane) [reply src/pages/ProfilePage.js]: @Mitchell139 you're correct.

**Review by bitcoinbrisbane — COMMENTED** [review]
Just what @Mitchell139 said

**Review by bitcoinbrisbane — APPROVED** [review]

**Review by JasminFarhi — APPROVED** [review]

**Pull request merged**